ocs_ci/deployment/fdf_standalone.py

```
@@ -0,0 +1,276 @@
+    """
+    catalog_image = config.DEPLOYMENT.get(
+        "fdf_standalone_catalog_image"
+    ) or config.DEPLOYMENT.get("ocs_registry_image", "")
+    if not catalog_image:
+        raise ValueError(
+            "A catalog image must be set for standalone FDF deployment. "
+            "Set 'fdf_standalone_catalog_image' or use the existing "
+            "'ocs_registry_image' config key (--ocs-registry-image CLI / "
+            "OCS_REGISTRY_IMAGE Jenkins parameter).\n"
+            "Example: 'cp.stg.icr.io/cp/df/isf-data-foundation-catalog:4.23.0-40'"
+        )
```

Ruff (0.16.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)